modules/contact/src/ContactMaster.C

```diff
@@ -417,6 +417,9 @@ contactFormulation(const std::string & the_name)
   else if ("augmented_lagrange" == name)
     formulation = CF_AUGMENTED_LAGRANGE;
 
+  else if ("tangential_penalty" == name)
```

Comment:
The compiler warns about `if (var = constant)` anyways...

So this is just my general opinion, but if there are two ways of doing a thing we should pick _one_ (that is actually what a style guide is all about!). If this way is better, change it _everywhere_. If the benefit is not that big (as it seems to me), stick to the de facto convention.